.github/workflows/package.yaml

```diff
@@ -46,6 +47,14 @@ jobs:
         persist-credentials: false
         path: rancher-desktop-docker-cli
 
+    - name: Set Docker CLI tag from workflow file
+      if: github.event_name == 'pull_request'
+      run: >
+        awk '/default: +v/ {print "DOCKER_CLI_REF=" $2}'
+        "${GITHUB_WORKSPACE}/rancher-desktop-docker-cli/.github/workflows/package.yaml"
```

Comment:
Would using [`${{ github.workflow_ref }}`](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) be correct here?

Also, it looks like [`yq`](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md) is preinstalled.

---

Actually, how is different from `${{ inputs.TAG }}`, since you're just looking for the default value? That also means that the _Set Docker CLI tag from tag_ step needs to happen __after__ _Set Docker CLI tag from inputs_ otherwise we're just always clobbering the ref?